quickstart/exampledata/config/http_pipeline.yml

```
@@ -1,17 +1,23 @@
 version: 1
+process_count: 2
 
 metrics:
-  enabled: true
+  enabled: false
   port: 8000
 
```

```suggestion
  enabled: true
  port: 8003
```

please add this port to prometheus scrape target in `quickstart/exampledata/config/prometheus/prometheus.yml` instead of the port `8000`

---

yep, I was disabling this because fda is already using this port and I wasn't sure what will happen when changing this port